aimarker.js

```
@@ -0,0 +1,838 @@
+            </div>
+          )}
+        </div>
+
+        <div className="bg-green-50 p-5 rounded-lg border border-green-100 relative overflow-hidden">
+          <div className="absolute top-0 right-0 p-2 opacity-10">
+            <CheckCircle className="w-16 h-16 text-green-600" />
+          </div>
+          <h4 className="text-xs font-bold text-green-700 uppercase tracking-wider mb-2">Model Answer</h4>
+          <div 
+            className="text-slate-800 font-serif text-sm leading-relaxed"
+            dangerouslySetInnerHTML={{ __html: feedback.rewrite }} 
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)